changelog.txt

```diff
@@ -3,6 +3,7 @@
 = 8.0.0 - xxxx-xx-xx =
 * Fix - Safeguards added to the Subscriptions Totals template used in the My Account area, to guard against fatal errors that could arise in unusual conditions.
 * Fix - Correctly load product names with HTML on the cart and checkout shipping rates.
+* Fix - Admin notices are now locked down to the current user by default.
```

Comment:
Offering an alternative changelog entry:

```suggestion
* Fix - Improve our admin notice handling to ensure notices are displayed to the intended admin user.
```

🤷

Reply:
Yes, that's nicer. Committed!